lib/oli_web/components/delivery/scored_activities/scored_activities.ex

```diff
@@ -0,0 +1,624 @@
+defmodule OliWeb.Components.Delivery.ScoredActivities do
+  use Surface.LiveComponent
```

Comment:
Can't we make this a `Phoenix.LiveComponent` instead?

Reply:
We can, sure! I decided to go with this approach because within this component I need to render the `PagedTable` component, which is a `Surface.Component`. 
Although we could render it, I do not like having to pass all the component props to make it work inside a Phoenix.LiveComponent. I find it more readable/clean this way, WDYT?

Reply:
Although I find it cleaner this way, if we want to stop using Surface I think it would be better to make it a LiveView component. We can leave it as is and change it later because it shouldn't be difficult to change it. I'll just leave it up to you to decide 😅

Reply:
makes sense, I will refactor it